### DISCUSSION TITLE
Removed issues

### DISCUSSION THREAD

**Discussion by sunderme** [texstudio-org/texstudio#1496] [General]
Does anybody have an idea why a number of issue reports have disappeared ?
I assume that the user closed his account but I would not expect issue reports to disappear with it...
Example: https://github.com/texstudio-org/texstudio/issues/1482
The annoying part is that the commit message relies on the issue being present to give details.

**Comment by muzimuzhi** [comment]
That's strange. According to [this GitHub Doc](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/deleting-your-user-account), 
> Deleting your user account [...] Issues and pull requests you've created and comments you've made in repositories owned by other users will not be deleted - instead, they'll be associated with our Ghost user.

And if my remember correctly, only maintainers can delete issues. Maybe you can ask this in https://github.community/ or contact the GitHub team directly, see https://github.com/contact.

**Reply by muzimuzhi** [reply]
Looks like every issue opened by JonathanLevine657 is removed. Here "JonathanLevine657" is the name shown in email titles, and it may or may not be the actual github user name.

**Comment by muzimuzhi** [comment]
Those removed issues are now back. As @JonathanLevine657 said in https://github.com/texstudio-org/texstudio/issues/1488#issuecomment-785257715,

> My account was flagged for no reason and it took 5 days to get Github to review, that's why my issues disappeared - Sorry, but it was out my control.
